lib/vsc/filesystem/posix.py

```diff
@@ -388,11 +389,12 @@ def populate_home_dir(self, user_id, group_id, home_dir, ssh_public_keys):
         authorized_keys = os.path.join(home_dir, '.ssh', 'authorized_keys')
         default_dsa_key = os.path.join(home_dir, '.ssh', 'id_dsa.pub')
         default_rsa_key = os.path.join(home_dir, '.ssh', 'id_rsa.pub')
+        default_ed25519_key = os.path.join(home_dir, '.ssh', 'id_ed25519.pub')
```

Comment:
make the key types a list and use that list to collect files and iterate below. So that in future you only need to add a key type to that list to have it fully working?